scripts/reboot

```diff
@@ -14,6 +14,16 @@ function stop_sonic_services()
     sleep 3
 }
 
+function clear_warm_boot()
+{
+    # If reboot is requested, make sure the outstanding warm-boot is cleared
```

Comment:
If we config warm_reboot flag in DB, deleting files are not needed.

Reply:
We do need to delete these back up files, because database.sh would restore them, therefore restore the warm boot flag. Which is what we don't want during recovery. Particularly, removing all prevents a partial restoration.